examples/checkout/sdk-load-widgets-with-nextjs/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@examples/sdk-load-widgets-with-nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "prepare:widgets": "mkdir -p tests/utils/local-widgets-js/ && cp $(yarn workspace @imtbl/checkout-widgets exec pwd)/dist/*.js tests/utils/local-widgets-js/",
```

Comment:
Is it possible to pull the checkout widgets js files from the output of the @imtbl/sdk build process rather than the output of the checkout-widgets build process, since only the @imtbl/sdk package is published and this would more closely resemble the what is actually exposed to customers?